src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj

```diff
@@ -15,9 +15,7 @@
     <Reference Include="Microsoft.JSInterop.WebAssembly" />
     <ProjectReference
       Include="..\..\WebAssemblyHttpHandler\src\Microsoft.AspNetCore.Components.WebAssembly.HttpHandler.csproj"
-      IncludeAssets="commit"
```

Comment:
Was this ever correct? From what I researched "commit" is not a valid value for "IncludeAssets"?

Reply:
Well, it was at some point. because I did this test this locally. I clearly introduced a typo at some point.